tests/robot-tests/tests/general_public/methodology_page_absence.robot

```diff
@@ -19,12 +19,12 @@ Navigate to Pupil absence in schools in England methodology page
 Should only be one link to absence methodology page
     [Documentation]  DFE-1359
     [Tags]  HappyPath
-    ${count}=  get element count  xpath://a[text()="Pupil absence statistics: methodology"]
+    ${count}=  get element count  xpath://ul/h3[text()="Pupil absence statistics: methodology"]
```

Comment:
Can you make use of the Python changes you already made for "page contains methodology link" here too?

Reply:
If I remember correctly, this test was added in response to a bug where the same methodology was appearing multiple times. I could add a check for the methodology being the only one to the python keyword, but I'm inclined to leave it as a separate test so it flags if that bug specifically reoccurs.